.github/workflows/lint.yml

```diff
@@ -87,7 +87,7 @@ jobs:
         run: cargo sort --workspace --check
 
       - name: Install cargo-udeps
-        run: cargo install cargo-udeps --version 0.1.53 --locked
+        run: cargo install cargo-udeps --version 0.1.57 --locked
```

Comment:
Not gonna lie, I expected this to be a bigger version jump considering it was broken for you locally